patches/kernel/4.1.6/0001-net-phy-tune-get_phy_c45_ids-to-support-more-c45-phy.patch

```diff
@@ -0,0 +1,60 @@
+From 583c128d1787b3146622215b2a804321ffb12d3d Mon Sep 17 00:00:00 2001
```

Comment:
You are posting 54 kernel patches to the 4.1.y kernel tree.  That is quite a lot :)

What is the upstream status of each of these patches?

The ONIE rebases to the current 4.1.y kernel every 3 months as part of the release process.  The general merge window will close in about 3 weeks and the rebasing process will start.  I am trying to reason about how much work it will to rebase all of this to 4.1.y (currently 4.1.10) at that time.  Is it likely that some of these are already in 4.1.10?